lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -224,14 +224,16 @@ def encounter_details(document: dict) -> dict:
         }
 
 
-def race(race_name: str) -> list:
+def race(races: str or list) -> list:
     """
-    Given a *race_name*, returns the matching race identifier found in Audere
-    survey data.
+    Given one or more *races*, returns the matching race identifier found in
+    Audere survey data.
     """
-    if race_name is None:
+    if races is None:
         LOG.debug("No race response found.")
         return [None]
+        
+    races = list(races)
```

Comment:
If `races` is a string, `list(races)` turns it into a list of characters.

Reply:
Yep I just caught that in local testing. I have fixed the code. Thanks!